src/main/webapp/javascript/user-page.js

```diff
@@ -1,6 +1,18 @@
 
 function openModal() {
   document.getElementById("recipe-modal").style.display = "block";
+  fetchBlobstoreUrl();
+}
+
```

Comment:
Comment documentation for your method

Reply:
it's not that I'm ignoring this comment, but I'm not exactly sure what this method does
it was in the blobstore API tutorial
I'll try to figure it out for my next PR